tests/test_applets/objectCounting/testObjectCountingGui.py

```diff
@@ -50,6 +48,10 @@
 logger.setLevel(logging.DEBUG)
 
 
+# Sample Sigma value for OpCounting.opTrain (OpTrainCounter).
+COUNTING_SIGMA = 4.2
```

Comment:
Not a huge fan of global variables, since it is used in one class only, you can make it a class variable in `setupClass`

Reply:
Got it, although I don't see any harm in a module-level constant used in a test module.